src/NServiceBus.Transport.RabbitMQ.TransportTests/NServiceBus.Transport.RabbitMQ.TransportTests.csproj

```diff
@@ -11,11 +11,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="[15.*,16)" />
-    <PackageReference Include="NServiceBus" Version="[7.0.0-*,8)" />
-    <PackageReference Include="NServiceBus.TransportTests.Sources" Version="[7.0.0-*,8)" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="NServiceBus" Version="7.0.0-beta0007" />
```

Comment:
Same for this package reference. NServiceBus.TransportTests.Sources should pull in the correct version.

Reply:
True, I'll remove it.